templates/insights/insights_index_page.html

```diff
@@ -10,6 +10,21 @@
         <h2 class="sr-only">{% trans "Each collection insight takes an in-depth look into a group of the records we hold." %}</h2>
     </div>
 
+    {% comment %}
+        NOTE: TO BE REFINED BY FE
+        {% if page.featured_insight %}
+            <div class="featured-insight">
+                <a href="{% pageurl page.featured_insight %}">
+                    <h3>
+                        <small>{% trans "Collection hightlight" %}</small><br>
+                        {{ page.featured_insight.title }}
+                    </h3>
+                    <p>Donec nec justo eget felis facilisis fermentum. Aliquam porttitor mauris sit amet orci. Aenean dignissim pellentesque felis.</p>
+                </a>
```

Comment:
Would it be possible to only wrap the `<h3>` in the `<a>` tag? Following an accessibility audit, we decided to make links shorter as long ones (e.g. links wrapping several sentences) were raised as a potential issue by the assessors.

Reply:
Hi @JamesWChan - This is completely throw-away code :) I was just dropping things into a comment here to help you understand what you're dealing with when you get to the FE